vm/systemSmartContracts/delegationManager.go

```diff
@@ -495,6 +499,90 @@ func (d *delegationManager) getContractConfig(args *vmcommon.ContractCallInput)
 	return vmcommon.Ok
 }
 
+func (d *delegationManager) claimMultipleDelegation(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	returnCode := d.executeFuncOnListAddresses(args, "claimRewards")
```

Comment:
constants for function names? `claimRewards`, `reDelegateRewards` & `delegate`

Reply:
done